participants/matin-cloud-pan.md

```diff
@@ -0,0 +1,63 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+*Project name is matin-cloud-pan , and our team is Matin Cloud (麦田云际) . Now , we are working on filecoin and IPFS*
+
+## (1) Brief Project Description
+
+*This project mainly stores various data, such as pictures, audio and video, etc.*
+
+## (2) Link to Project UI
+
+*Web: http://pan.mtfs.top*
+
+*Android & iOS: https://app.mtfs.top/down/pan-down/downpage.html*
+
+## (3) What does your application/UI do?
+
+*Provide individuals with secure data storage services on filecoin, and the live broadcast function will be developed in the future.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+* *
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will collect data such as videos, audios, pictures, etc. We have the power to process the data, and the data does not contain any sensitive or illegal information.*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*Maybe 3TB-5TB , or more !!*
+
+## (7) How are you structuring the data?
+
+*We collect image and video data(unstructured data), and manage these data sets and our application data through MySQL/Mongodb database.*
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*We will store each file (or file package) on the network, and save the information record and index in a relational database for query. The size of the storage transaction is around 3GB, or less.*
+
+## (9)  What tech stack will you use to store the data?
+
+*Lotus*
```

Comment:
Powergate has both a lotus node and ipfs node as part of its implementation which may be helpful for your use-case.

Reply:
Powergate yes！！！Maybe we will use Powergate for this project, and now we are considering whether it is really suitable for our current project！（We are currently implementing a solution using ipfs for caching and lotus for cold storage of data, and will soon apply it to this application. This scheme also draws on the design ideas of Powergate）